packages/x-adapter-platform/src/types/requests/request.types.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Interface for any `request` with `extra parameters`.
+ *
+ * @public
+ */
+export interface PlatformExtraParamsRequest {
```

Comment:
Export only the needed interfaces:

export interface PlatformPopularSearchesRequest
  extends PlatformPageableRequest,
    PlatformExtraParamsRequest {}

Reply:
Sure, that makes sense. Thanks!